src/cdk/schematics/ng-update/data/input-names.ts

```diff
@@ -14,8 +14,11 @@ export interface InputNameUpgradeData {
   replace: string;
   /** The new name for the @Input(). */
   replaceWith: string;
-  /** Whitelist where this replacement is made. If omitted it is made in all HTML & CSS */
-  whitelist: {
+  /**
+   * Controls which file types in which this replacement is made. If omitted, it is made in all
+   * files.
+   */
+  fileTypeFilter: {
```

Comment:
This doesn't seem about file types for this data set. I can't come up with a good name either, but maybe something around `limitedTo: {attributes: ['a', 'b']}`

This comment also applies to similar data sets where we don't filter by file type. I think `fileTypeFilter` is not 100% accurate either as the options are HTML, stylesheet and strings. Obviously "strings" is a bit different here. Maybe we can change that to something better, or change the option name (as proposed with the comment below)

Reply:
Done, I like `limitedTo` as it reads nicely in dot notation.